Tasks/AppStorePromote/app-store-promote.ts

```diff
@@ -21,10 +22,39 @@ export class UserCredentials {
     /* tslint:enable:no-empty */
 }
 
+/**
+ * Information for the App Store Connect API key used by fastlane
+ * See https://docs.fastlane.tools/app-store-connect-api/#using-fastlane-api-key-json-file
+ */
+export interface ApiKey {
+    /**
+     * Key ID (for example 'D383SF740')
+     */
+    key_id: string;
+    /**
+     * Issuer ID (for example '6053b7fe-68a8-4acb-89be-165aa6465141')
+     */
+    issuer_id: string;
+    /**
+     * The base64-encoded private key contents of the p8 file from Apple.
+     */
+    key: string;
+    /**
+     * Optional, set to true to use Enterprise account
+     */
+    in_house?: boolean;
+    /**
+     * Indicates whether the key content is base64 encoded
+     */
+    is_key_content_base64: boolean;
+}
+
 async function run() {
     const appSpecificPasswordEnvVar: string = 'FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD';
     const fastlaneSessionEnvVar: string = 'FASTLANE_SESSION';
+    let apiKeyFileName: string = undefined;
```

Comment:
I would suggest changing the variable name to `apiKeyFile` or `apiKeyFilePath` since it's actually a path.

Reply:
Great point, I'll fix it.

Reply:
@egor-bryzgalov Fixed in https://github.com/microsoft/app-store-vsts-extension/pull/189/commits/b79c37a1495a08eaf00671b879017554ac815080